activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -0,0 +1,20 @@
+---
+type: Resource
+bpmn: lifecycle.bpmn
+---
+
+
+# Criteria  for codebase stewardship
+
+Here are the criteria the staff of the Foundation for Public Code will use to help evaluate whether a codebase is ready for/conducive to incubation. They are used before and after the 'Assess' phase in the codebase stewardship lifecycle. 
+
+* Business case(s): what is/are the business case(s) for the codebase becoming public code and being reused?
+* Co-develop-ability: does the codebase lend itself to collaborative development?
+* Community: does the codebase have an active and engaged community?
+* Community potential: is there a strong potential for the community to grow beyond its existing members?
+* Code reusability: is the code written in a way that makes it easy to be reused? 
+* Reuse potential: to what extent can the codebase be reused, both in a different context or for a different purpose?
+* Societal impact: what is the potential societal impact of the codebase becoming public code and being reused?
+* Story: what is the story-telling and communication value of this codebase becoming public code and being reused?  
+
+Note: These are not ordered by importance.
```

Comment:
I'd forgo this note, if they were ordered they would be numbered.

Reply:
I'd keep it - because I don't think that assumption holds for everyone. Have moved it to top so it's not a note.